timur/webpack.config.js

```diff
@@ -1,74 +1,79 @@
 
-module.exports = env => ({
-  context: path.resolve(__dirname, 'lib/client'),
+module.exports = (env) => ({
+  context: path.resolve(__dirname, "lib/client"),
   resolve: {
-    extensions: [ '.js', '.jsx', '.scss', '.png', '.jpg', '.jpeg', '.svg' ],
+    extensions: [".js", ".jsx", ".scss", ".png", ".jpg", ".jpeg", ".svg"],
     alias: {
-      'code-mirror': path.join(__dirname, 'node_modules/codemirror/lib'),
-      'react-table': path.join(__dirname, 'node_modules/react-table')
-    }
+      "code-mirror": path.join(__dirname, "node_modules/codemirror/lib"),
+      "react-table": path.join(__dirname, "node_modules/react-table"),
+    },
   },
-  entry: ['./jsx/timur.jsx', './scss/application.scss' ],
+  entry: ["./jsx/timur.jsx", "./scss/application.scss"],
   output: {
-    filename: 'public/js/timur.bundle.js',
-    path: __dirname
+    filename: "public/js/timur.bundle.js",
+    path: __dirname,
+    publicPath: "/js/",
   },
   module: {
     rules: [
       {
-        loader: 'babel-loader',
+        loader: "babel-loader",
 
         // Skip any files outside of your project's `src` directory
-        include: [
-          path.resolve(__dirname, 'lib/client/jsx'),
-        ],
+        include: [path.resolve(__dirname, "lib/client/jsx")],
 
         // Only run `.js` and `.jsx` files through Babel
         test: /\.jsx?$/,
 
         // Options to configure babel with
         query: {
-          presets: ['env', 'react'],
-          plugins: ['transform-object-rest-spread']
-        }
+          presets: ["env", "react"],
+          plugins: ["transform-object-rest-spread"],
+        },
       },
 
       {
         test: /\.(jpe?g|png|svg)$/i,
 
-        include: [
-          path.resolve(__dirname, 'lib/client/img'),
-        ],
+        include: [path.resolve(__dirname, "lib/client/img")],
 
-        loader: 'file-loader',
+        loader: "file-loader",
 
         options: {
-          name: '[name].[ext]',
-          outputPath: 'public/images/',
-          publicPath: '/images'
-        }
+          name: "[name].[ext]",
+          outputPath: "public/images/",
+          publicPath: "/images",
+        },
       },
 
       {
         // sass / scss loader for webpack
         test: /\.(sass|scss)$/,
 
-        loader: ExtractTextPlugin.extract(['css-loader', 'sass-loader'])
-      }
-    ]
+        loader: ExtractTextPlugin.extract(["css-loader", "sass-loader"]),
+      },
+    ],
   },
   plugins: [
-    new ExtractTextPlugin({ // define where to save the file
-      filename: 'public/css/timur.bundle.css',
+    new ExtractTextPlugin({
+      // define where to save the file
+      filename: "public/css/timur.bundle.css",
       allChunks: true,
     }),
     new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify(env.NODE_ENV),
-      }
-    })
+      "process.env": {
+        NODE_ENV: JSON.stringify(env ? env.NODE_ENV : "development"),
+      },
+    }),
+    new EventHooksPlugin({
```

Comment:
We don't need this part for timur, this is metis only for the zip download part.  Clearing this will fix the tests.

Reply:
Oh wait, you need it for copying the routes piece?
Ah, so the file needs a `fs` import at top and the filename part needs to be changed.

Reply:
No, you're right, I'll take this out. I added this because I thought I needed it to copy the `routes` file, but I don't think that's actually necessary.